src/lib/connectTraceToPlot.js

```diff
@@ -71,7 +79,14 @@ export default function connectTraceToPlot(WrappedComponent) {
     }
 
     render() {
-      return <WrappedComponent name={this.name} {...this.props} />;
+      return (
+        <WrappedComponent
+          name={this.name}
+          type={this.type}
+          plotIcons={this.plotIcons}
```

Comment:
What if the wrapped component isn't a `<Fold>` ?

Reply:
That could be ok so long as we have a consistent API for accepting icons in our components. Then any `icon` aware component can be swapped in here are it'll work. Do we have any other components that take `icon` as props? If so lets get the naming standardized.

Reply:
```
➜  src git:(TemporaryFixForAxesTypeClearing) ✗ ack PropTypes | grep icon
components/containers/Panel.js:66:  icon: PropTypes.node,
components/containers/MenuPanel.js:58:  iconClass: PropTypes.string,
components/widgets/Button.js:41:  icon: PropTypes.any,
components/widgets/RadioBlocks.js:72:      icon: PropTypes.string,
```

Reply:
MenuPanel takes an `iconClass`. Seems like it needs to be updated to our new pattern.

Reply:
it looks like `MenuPanel` isn't using iconClass at all, we can probs just remove it.